pycon_archive_past_website/websites/classes.py

```diff
@@ -0,0 +1,115 @@
+"""Crawler related classes"""
```

Comment:
How about naming this folder and file as `crawlers/base.py`? `websites` and `classes` seem to be too general.

Reply:
Updated.